scripts/i18n/PROGRESS.template.md

```diff
@@ -0,0 +1,23 @@
+## For New Translators
+
+To translate a page:
+
+1. Check that no one else has claimed your page in the checklist and comments below.
+2. Comment below with the name of the page you would like to translate. **Please take only one page at a time**.
+3. Clone this repo, translate your page, and submit a pull request!
+
+Before contributing, read the glossary and style guide to understand how to translate various technical and Gatsby-specific terms.
```

Comment:
```suggestion
Before contributing, read the [Glossary](https://www.gatsbyjs.org/docs/glossary) and [Style Guide](https://www.gatsbyjs.org/contributing/gatsby-style-guide/) to understand how to translate various technical and Gatsby-specific terms.
```

Is this correct, or should we add clarifying language about where to find those materials?

Reply:
Good point -- it should refer to the style guide for the translation itself. I'll add a link.